src/components/VmsList/Vms.js

```diff
@@ -1,25 +1,69 @@
 import React from 'react'
+import ReactDOM from 'react-dom'
 import PropTypes from 'prop-types'
+import { Link } from 'react-router-dom'
+import { Icon, Button } from 'patternfly-react'
 
 import { connect } from 'react-redux'
 
 import style from './style.css'
+import sharedStyle from '_/components/sharedStyle.css'
 import Vm from './Vm'
 import Pool from './Pool'
 import ScrollPositionHistory from '../ScrollPositionHistory'
 import { getByPage } from '_/actions'
 import { filterVms, sortFunction } from '_/utils'
+import { msg } from '_/intl'
 import InfiniteScroll from 'react-infinite-scroller'
 import Loader, { SIZES } from '../Loader'
 
+const VmsSettingsButton = ({ checkedVms = [] }) => {
+  const container = document.getElementById('vm-settings-btn-box')
+  if (container) {
+    return ReactDOM.createPortal(
+      <Link to={`/vms-settings/${checkedVms.join('/')}`} className={`btn btn-default ${sharedStyle['settings-icon']}`} disabled={checkedVms.length === 0}>
```

Comment:
This renders as an `a` tag ultimately so the disabled prop shows it disabled but does NOT prevent the click from firing the action and killing the app (since no VMs have been selected, errors out to the global error page)